cmd/juju/romulus/listplans/list_plans.go

```diff
@@ -71,12 +72,19 @@ func (c *ListPlansCommand) Info() *cmd.Info {
 // Init reads and verifies the cli arguments for the ListPlansCommand
 func (c *ListPlansCommand) Init(args []string) error {
 	if len(args) == 0 {
-		return errors.New("missing arguments")
+		return errors.New("missing charm-store charm URL argument")
 	}
 	charmURL, args := args[0], args[1:]
 	if err := cmd.CheckEmpty(args); err != nil {
 		return errors.Errorf("unknown command line arguments: " + strings.Join(args, ","))
 	}
+	curl, err := charm.ParseURL(charmURL)
+	if err != nil {
+		return errors.Annotatef(err, "unable to parse charm URL")
+	}
+	if !charm.CharmStore.Matches(curl.Schema) {
+		return errors.Errorf("charm-store charm URLs are only supported")
```

Comment:
The error message wording looks a bit strange to me. How about:

```suggestion
		return errors.NotSupportedf("non charm-store URLs")
```